sklearn/multioutput.py

```diff
@@ -474,6 +474,40 @@ def predict_proba(self, X):
         results = [estimator.predict_proba(X) for estimator in self.estimators_]
         return results
 
+    def _check_decision_function(self):
```

Comment:
Since we have the same pattern in `_check_predict_proba` and `_check_decision_function`, I think that we should have a single function `_check_has_response_method` that would take the name of the method to be checked (i.e. `predict_proba` or `decision_function`.

Reply:
We should add this to all the `fit`, `predict` and `predict_proba` methods.